### PR TITLE
Adding Spanish, Portuguese and Russian SDA hymns

### PR DESCRIPTION
**Pull request by Burkifa23**
es for Spanish: Himanrio Adventista. This will be beneficial to those in South America, Spain, and the USA.
pt for portuguese: Hinàrio Adventista Do Sétiomo Dia. This will be beneficial to Angola, Brazil, Cape Verde, East Timor, Guinea Bissau, Macau, Mozambique, Portugal, & São Tomé and Príncipe.
ru for Russian:  Гимн адвентистов седьмого дня

Files that are still a work in progress include:
Kirundi is from Burundi.
dg and hl are French Hymns. French is spoken throughout Africa and Europe.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -76,7 +76,37 @@
     },
     {
         "key": "kinyarwanda",
-        "title": "Indirimbo Zo Guhimbaza Iman",
+        "title": "Indirimbo Zo Guhimbaza Imana",
         "language": "Kinyarwanda"
+    },
+    {
+        "key": "hl",
```
Comment (TinasheMzondiwa) [inline config.json:83]: Since this is still WIP let's add it to the config when its ready

**Comment by OlivierJM** [comment]
@TinasheMzondiwa @Burkifa23 Shouldn't the content of the file have a structure like this 

```
{
          "title": "3 Face To Face",
          "number": 3,
          "content": "html or markdown"
}
```

I have noticed that files in this PR have a markdown property instead content.

**Comment by TinasheMzondiwa** [comment]
The apps use a different view to render markdown and another one to render html so it's easier to just define these as different fields.

**Comment by OlivierJM** [comment]
> The apps use a different view to render markdown and another one to render html so it's easier to just define these as different fields.

Makes sense.

**Comment by OlivierJM** [comment]
Maybe I should modify the renderer I worked on, to handle different field names.

File: config.json [hunk 2]
```diff
@@ -76,7 +76,37 @@
     },
     {
         "key": "kinyarwanda",
-        "title": "Indirimbo Zo Guhimbaza Iman",
+        "title": "Indirimbo Zo Guhimbaza Imana",
         "language": "Kinyarwanda"
+    },
+//    {
```
Comment (TinasheMzondiwa) [inline config.json:82]: Just delete these then we can merge

**Review by TinasheMzondiwa — APPROVED** [review]

**Pull request merged**